main.go

```diff
@@ -52,14 +52,15 @@ func save() {
 	}
 }
 
-func loadConfiguration() (cfg *config.Config, err error) {
-	customConfigFile := os.Getenv("GATUS_CONFIG_FILE")
-	if len(customConfigFile) > 0 {
-		cfg, err = config.Load(customConfigFile)
-	} else {
-		cfg, err = config.LoadDefaultConfiguration()
+func loadConfiguration() (*config.Config, error) {
+	var configPath = os.GetEnv("GATUS_CONFIG_PATH")
+
+	// backwards compatibility
+	if len(configPath) == 0 {
+		configPath = os.GetEnv("GATUS_CONFIG_FILE")
```

Comment:
```suggestion
func loadConfiguration() (*config.Config, error) {
	var configPath = os.Getenv("GATUS_CONFIG_PATH")
	// Backwards compatibility
	if len(configPath) == 0 {
		configPath = os.Getenv("GATUS_CONFIG_FILE")
```

Reply:
I was going to commit it myself, but I think you have "allow edits from maintainer" unchecked 😔